_drafts/SC3-1-2-lang.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Validity of HTML Attribute with in elements of page body
+
+description:
+- This rule checks that `lang` and/or `xml:lang` attribute on elements with in the `body` of a web page are correct.
+
+success_criterion:
+- 3.1.2
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Bryn Anderson
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+- Any DOM element, other than the root DOM node with a `lang` and/or `xml:lang` attribute.
+
+### Expectation
+
+- The value of the `lang` and/or `xml:lang` conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+This rule is only applicable in a scenario where assistive technologies are inconsistent about how they determine the language of the page.
```

Comment:
I don't think we should use the term "is only applicable" here, to not confuse this with the Applicability section of the rule.

I suggest something more along these lines: "Most assistive technologies will rely solely on the `lang` attribute for determining the language of the page. The requirement tested in this rule will only give rise to accessibility issues in cases where assistive technologies are inconsistent about how they determine the language of the page."